addons/crm/crm_lead.py

```diff
@@ -1052,64 +984,62 @@ def message_update(self, cr, uid, ids, msg, update_vals=None, context=None):
                 key = maps.get(res.group(1).lower())
                 update_vals[key] = res.group(2).lower()
 
-        return super(crm_lead, self).message_update(cr, uid, ids, msg, update_vals=update_vals, context=context)
+        return super(CrmLead, self).message_update(msg, update_vals=update_vals)
 
     # ----------------------------------------
     # OpenChatter methods and notifications
     # ----------------------------------------
 
-    def schedule_phonecall_send_note(self, cr, uid, ids, phonecall_id, action, context=None):
-        phonecall = self.pool.get('crm.phonecall').browse(cr, uid, [phonecall_id], context=context)[0]
+    @api.multi
+    def schedule_phonecall_send_note(self, phonecall_id, action):
+        phonecall = self.env['crm.phonecall'].browse([phonecall_id])[0]
         if action == 'log':
```

Comment:
passed phonecall_id in list in browse and then used [0] . why ?
pass phonecall_id without list in browse.